examples/react-native-vanilla/__tests__/__snapshots__/storyshots.js.snap

```diff
@@ -114,7 +114,7 @@ exports[`Storyshots Knobs with knobs 1`] = `
     allowFontScaling={true}
     ellipsizeMode="tail"
   >
-    My name is Storyteller, I'm 70 years old, and my favorite fruit is apple.
+    My name is Storyteller, I'm 70 years old, and my favorite fruit is apple. I also enjoy watermelon.
```

Comment:
Tests are failing because of this snapshot, it was committed but there is not a story that changed.

Reply:
It was already changed in the other commit.